exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/orderedpartitioner/OrderedPartitionRecordBatch.java

```diff
@@ -23,6 +23,7 @@
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.TimeUnit;
 
+import org.apache.calcite.rel.RelFieldCollation.Direction;
```

Comment:
I'd prefer to avoid any formatting changes including `import` reorder unless a method, a class or a file is modified to include a fix. I don't see anything being fixed in `OrderedPartitionRecordBatch.java` except for `final` declarations.

Reply:
I'll back out the changes my editor made. I'll redo the changes with the current project options turned off. Stay tuned.

Reply:
Turned out this file was the wrong one: this code is never used. Should have fixed the `PartitionSenderTemplate` rather than this one. Reverted changes.